sample/common/src/jvmMain/kotlin/com/example/sqldelight/hockey/data/createDb.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.sqldelight.hockey.data
+
+import com.example.sqldelight.hockey.HockeyDb
+import com.squareup.sqldelight.db.SqlDriver
+import java.lang.IllegalStateException
+
+var driver: SqlDriver? = null
```

Comment:
oh boy

Reply:
Did a minor reformat to lateinit. Not "better", but visually looks betterish.